indexer/src/main/java/juice/modules/KafkaConsumerModule.java

```diff
@@ -0,0 +1,34 @@
+package juice.modules;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Provides;
+import config.ConfigurationFactory;
+import config.ConsumerConfiguration;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.common.serialization.IntegerDeserializer;
+import org.apache.kafka.common.serialization.StringDeserializer;
+
+import java.util.Properties;
+
+public class KafkaConsumerModule extends AbstractModule {
+
+    private final String fileUrl;
```

Comment:
Let's rename all `fileUrl` occurrences to ...FilePath, i.e.

```suggestion
    private final String kafkaConsumerFilePath;
```